exercises/node-javascript/add-esdk-start/retrieve.js

```diff
@@ -0,0 +1,17 @@
+/* ADD-ESDK-START:
```

Comment:
@mattsb42-aws @lizroth Here is the start retrieve file.  The ESDK needs to be added.